pkg/cloud/metadata/env.go

```diff
@@ -6,6 +6,7 @@ type ENVMetadata struct{}
 
 var MetadataEnv = map[MetadataKey]string{
 	RegionID: "REGION_ID",
+	AliUID:   "ALI_UID",
```

Comment:
```suggestion
	AliUID:   "ALIBABA_CLOUD_ACCOUNT_ID",
```
to be consistent with the official env for access key, etc.

BTW, can we get the UID from the access key? https://next.api.aliyun.com/api/Sts/2015-04-01/GetCallerIdentity